java-strings-2/src/test/java/com/baeldung/string/changecase/ToUpperCaseUnitTest.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.string.changecase;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Locale;
+
+import org.junit.Test;
+
+public class ToUpperCaseUnitTest {
+
+    private static final Locale TURKISH = new Locale("tr");
+    private String name = "John Doe";
+    private String foreignUppercase = "\u0069";

```

Comment:
Shouldn't this be called foreignLowercase?

Reply:
Yes, you are right, it should be `foreignLowercase`. I have made the correction and pushed in the updated test cases.